Index: OOPLAB/GameObjects/Animals.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/OOPLAB/GameObjects/Animals.cs b/OOPLAB/GameObjects/Animals.cs
--- a/OOPLAB/GameObjects/Animals.cs	
+++ b/OOPLAB/GameObjects/Animals.cs	
@@ -10,7 +10,7 @@
     class Animals : GameObject
     {
         private int _satiety;
-        private static int _maxSatiety;
+        private int _maxSatiety;
         int Satiety 
         {
             get { return _satiety; } 
